packages/client/hmi-client/src/components/workflow/tera-model-workflow-wrapper.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-	<tera-model :project="props.project" :asset-id="props.node.state.modelId" />
+	<tera-model
+		:project="props.project"
+		:asset-id="props.node.state.modelId"
+		@new-model-configuration="refreshModelNode"
+		@update-model-configuration="refreshModelNode"
```

Comment:
If both do the same job, why having two different emit?

Reply:
I thought of merging the two events, but I thought it would bring-about more confusion because typically update and create are distinct events.